src/EditorFeatures/Test/Completion/AbstractCompletionProviderTests.cs

```diff
@@ -51,35 +51,67 @@ protected static bool CanUseSpeculativeSemanticModel(Document document, int posi
             return !service.GetMemberBodySpanForSpeculativeBinding(node).IsEmpty;
         }
 
+        internal static CompletionList GetCompletionList(CompletionListProvider provider, Document document, int position, CompletionTriggerInfo triggerInfo)
+        {
+            var itemsBuilder = ImmutableArray.CreateBuilder<CompletionItem>();
+            Action<CompletionItem> addCompletionItem = item =>
```

Comment:
great catch! Yes, it's slightly more cryptic but I think it's perfectly reasonable.